File: frontend/src/app/_shared/models/pessoa/pessoa.ts
import { Endereco } from "./endereco";

export class Pessoa {
    constructor(
        public id?: number,
        public nome?: string,
        public sexo?: string,
        public idade?: number,
        public endereco?: Endereco,
        public dataNasc?: Date,
        public cpf?: string,
        public telefone?: string,
        public senha?: string
    ) {}
}

File: frontend/mobile/app/src/main/java/com/tecno/aging/domain/models/pessoa/Pessoa.kt
package com.tecno.aging.domain.models.pessoa

open class Pessoa(
    open val nome: String = "",
    open val cpf: String = "",
    open val telefone: String = "",
    open val sexo: String = "",
    open val dataNasc: String = "",
    open val senha: String = "",
    open val endereco: Endereco = Endereco(),
    open val idade: Int
)